### ISSUE TITLE
Windriver can't get access to local temp directory

### ISSUE THREAD

**Issue by stefdaems** [imsardine/winappdriver#19]
Whenever I try to download an app from store using the winappdriver, I'm getting the message that the webdriver is not allowed to access the folder. 

I've been trying to redirect it to another directory on which all rights are open, but no success. Event when the webdriver is started with admin rights nothing changes.

![appnoaccess](https://cloud.githubusercontent.com/assets/12783064/9383096/902faff8-4746-11e5-9108-93718500a0f3.png)

**Comment by stefdaems** [comment]
I seems the ActivateStoreApp couldn't be found, fixed this by storing the ActivateStoreApp.exe file in the same directory as the winappdriver.

Can be closed as config issue

**Comment by imsardine** [comment]
@stefdaems What do you mean by "download an app from store", may I know capabilities you specified? According to the logs you provided, the error is irrelevant to app activation.

**Comment by stefdaems** [comment]
If in "app" capability I'm providing the URL towards the app in the play store. Am I right to say the winappdriver will download the app from the store and install it to the pc?

**Comment by imsardine** [comment]
Yes, WinAppDriver will download the installation package and install it.

However, it is expected that the installation package is a zip file. 

```
desired_caps = {
    'platformName': 'WindowsModern',
    'app': 'http://your-build-server/path/to/your/pacakge.zip',
    'packageName': 'your-app-package-name',
}
```

I don't know there exists a URL pointing to a package on Window Store. May I know the desired capabilities you specified, so I can try to reproduce the error.

**Comment by stefdaems** [comment]
Hi,

I'm not able to get a win8 app dowloaded from the store working with the webdriver on windows10.

I download the yelo play app and start webdriver. The webdriver starts the app correctly the first time, when a the webdriver is requested to open the app.

Whenever I close the app and webdriver and restart the webdriver I reopen the app via the webdriver but then the app crashes. When I try to open the app directly it doesn't open anymore.

Any idea?

Kind regards,
Stef

Verstuurd vanaf mijn iPhone
